pkg/cmd/roachtest/drop.go

```diff
@@ -74,7 +108,43 @@ gc:
 '`
 			run(stmtZone)
 
-			// TODO(tschottdorf): assert that the disk usage drops to "near nothing".
+			// We're waiting a maximum of 5 minutes to makes sure that the drop operations clear the disk.
+			for i := 0; i < 5; i++ {
+				allNodesSpaceCleared := true
+				for j := 1; j <= nodes; j++ {
+					out, err := c.RunWithBuffer(ctx, c.l, j, fmt.Sprintf("du -sc {store-dir:%d}", j))
+					if err != nil {
+						return err
+					}
+
+					str := string(out)
+
+					// We need this check because sometimes the first line of the roachprod output is a warning
+					// about adding an ip to a list of known hosts.
+					if strings.Contains(str, "Warning") {
+						str = strings.Split(str, "\n")[1]
+					}
+					result := strings.Split(str, "/")
```

Comment:
I thought you would just split by whitespace. The output is something like the below, right? You just get lucky that your store-dir is absolute.

```
7070616	some/path/cockroach
```